tests/test_pytorch_distributed.py

```diff
@@ -0,0 +1,287 @@
+from __future__ import annotations
+
+import datetime
+import os
+
+import pytest
+
+import optuna
+from optuna._imports import try_import
+from optuna.testing.pruners import DeterministicPruner
+from optuna.testing.storages import StorageSupplier
+
+from optuna_integration import TorchDistributedTrial
```

Comment:
This line causes the following error.
```sh
tests/test_pytorch_distributed.py:13: in <module>
    from optuna_integration import TorchDistributedTrial
E   ImportError: cannot import name 'TorchDistributedTrial' from 'optuna_integration'
```

Please use `from optuna_integration.pytorch_distributed import TorchDistributedTrial` instead.